### DISCUSSION TITLE
NodeRed flow to control the Lay-Z-Spa via NodeRed UI

### DISCUSSION THREAD

**Discussion by zolonn** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#617] [General]
hi all,

here is my flow to control the spa using NodeRed's biuld-in UI (node-red-dashboard)
UI is build on S100101 6-wire pump 

/BR
ZoloN

[LayZSpa-control_via_NodeRed_UI.json.txt](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/files/12474773/LayZSpa-control_via_NodeRed_UI.json.txt)

**Comment by Captain0Kangaroo** [comment]
I've just started playing with Node Red - Imported and connected to my MQTT but i've no idea what to do next :-)

**Reply by zolonn** [reply]
1st of all, if you get errors while importing, you have to install the node-red-dashboard (hamburger menu -> manage palette -> install)
then reconfigure the MQTT nodes for your broker
when all done, you can access the dashboard under http://<nodered URL><:1880>/ui (look the docs here: https://flows.nodered.org/node/node-red-dashboard)
